components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -457,7 +457,7 @@
     public static final String CERTIFICATE_INFORMATION = "CertificateInformation";
     public static final String AUTHORIZATION_HEADER_DEFAULT = "Authorization";
     public static final String HEADER_TENANT = "xWSO2Tenant";
-    public static final String X_WSO2_TENANT_HEADER = "X-WSO2-Tenant";
+    public static final String X_WSO2_TENANT_HEADER = "XWSO2Tenant";
```

Comment:
Do we need this change? :thinking:

Reply:
this header was read from ExtendedISAUthHandler therefore this header can't change

Reply:
Yeah. Reverted the change. Got a 401 error earlier, but that was with an old IS connector hosted in docs. Created [1] to updated the docs with latest connector.

[1] https://github.com/wso2/docs-apim/issues/3442